### PR TITLE
Update KMW copy.

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #7484 

## Relevant technical choices

Updated copy for KMW in locations noted in task IB. Changed copy in the 1 present test for the widget content above CTA. 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/user-input/UserInputQuestionnaire.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default function UserInputQuestionnaire() {
 						'google-site-kit'
 					) }
 					description={ __(
-						'Based on your answers, Site Kit will tailor the metrics and advice you see on your dashboard to help you make progress in these areas',
+						'Based on your answers, Site Kit will tailor the metrics and recommendations you see on your dashboard to help you make progress in these areas.',
```
Comment (hussain-t) [inline assets/js/components/user-input/UserInputQuestionnaire.js:251]: We have the period (`.`) at the end of the `purpose` and `goals` modified description. Can we add it to the `postFrequency` step as well?

`Based on your answer, Site Kit will suggest new features for your dashboard related to content creation.`

File: assets/js/components/user-input/UserInputPreviewGroup.js [hunk 2]
```diff
@@ -211,7 +211,10 @@ export default function UserInputPreviewGroup( {
 
 							<div className="googlesitekit-user-input__preview-actions">
 								<SpinnerButton
-									disabled={ ! hasSettingChanged }
+									disabled={
+										! hasSettingChanged ||
+										values.length === 0
+									}
```
Comment (hussain-t) [inline assets/js/components/user-input/UserInputPreviewGroup.js:217]: Why was this condition added?

**Review by hussain-t — CHANGES_REQUESTED** [review]
Thanks, @zutigrm. Overall, the changes look good. I have left a couple of nitpicky comments. Please take a look.

We don't need the last point of the QAB, as the QA engineers won't check the code changes or perform the testing in the local environment. You can remove this:

`Verify that changed files containing the content are wrapped with i18n function.`

File: assets/js/components/user-input/UserInputPreviewGroup.js (hunk 2 above)
Comment (zutigrm) [reply assets/js/components/user-input/UserInputPreviewGroup.js]: Thanks for spotting it. I mistakenly included/mixed it from other task. Removed.

**Review by hussain-t — APPROVED** [review]
Nice one, @zutigrm. LGTM 👍

**Review by tofumatt — APPROVED** [review]

**Pull request merged**